io.c

```diff
@@ -9446,8 +9452,10 @@ rb_io_set_encoding_by_bom(VALUE io)
  *
  *  Examples:
  *
- *    File.new('t.tmp', autoclose: true)
- *    File.new('t.tmp', internal_encoding: nil)
+ *    f = File.new('t.tmp', autoclose: true)
```

Comment:
I think `autoclose` eliminates the need for `f.close`?

Reply:
Omitted.